### ISSUE TITLE
Possible Q595 enhancement

### ISSUE THREAD

**Issue by awolfe76** [cfpb/hmda-platform-ui#40]
@debseidner - want to start gathering your thoughts about this here?

**Comment by debseidner** [comment]
What about if we displayed a map of all the MSAs an institution is lending in and all the MSAs that they have branches in as an overlay (maybe in place of the Q595)? We could also maybe compare last year's to this years (or several previous year's to this year's). We could move the branch confirmation up front to the sign-in area or has to be done before you start filing for the year. Not sure but I think the map could be a cool way to display this.

To replace the Q595, we might actually consider moving the branch verification process up front - when an institution registers at the beginning of their filing, the institution should verify the branches that we have on file - the large institutions could upload branch information by CSV as the list may be too long to verify. We need to flush this out but definitely a possibility to move this up front with bank registration. I think from a data collection process, FRB does this now just during the edits so I would think this would be ok in terms of PRA or adding burden.

**Comment by debseidner** [comment]
Some sort of portfolio view for each institution that encompasses the following fields (like a pie chart or other representation of what their portfolio looks like and let them tell us why in each part or whether this seems reasonable)

property type
purpose of loan
loan type
action taken
purchaser type

Maybe a separate display of the breakdown of action takens. There are a lot of macros about if you have this action taken, it can't be more than x% of your portfolio, and if you have this other action taken then it can't be more than x% of your portfolio

Separately a study of preapproval and then how that corresponds to action taken when action taken is a preapproval (7 - preapproval request denied, 8 - preapproval request approved)

Another portfolio makeup diagram of HOEPA loans, action taken, rate spread and lien status

**Comment by debseidner** [comment]
We should have some chart breaking down GMI information (sex/race/ethnicity). There are quite a few macros focused on property type, action taken and sex/race/ethnicity.

**Comment by Kibrael** [comment]
@debseidner I'm working on building the data structures to support those visualizations. Can you build a list of the ones you need for loan portfolio displays? It'd be super helpful to have a list of what needs to be made.

I'm also unclear what need we're meeting with displaying the list of branches for verification. Panel requires a list of branch locations in order to make a lender list. We have two data sources (possibly soon to be 3 or more) for this. If we collect updated information, will we store it? Will we ship it back to primary data sources? Does the data we collect become a source of truth over NIC or SOD? Maybe we can talk about this during standup sometime.

**Comment by debseidner** [comment]
Edits doc on FFIEC site - http://www.ffiec.gov/hmda/pdf/edit2016.pdf

**Comment by awolfe76** [comment]
Removed a few comments from here and created #41.

**Comment by awolfe76** [comment]
This is on hold waiting for a larger conversation during HMDA regroup, April 11-20. We can update after.

**Comment by debseidner** [comment]
@Kibrael I think the idea would be to update NIC with the updated branches to just confirm that we have the right ones in our search for whether entities should file and also are providing MSA/MD data for MSA/MDs in which they have branches. After meeting with Jason and Feng, I will need to update this with some different suggestions for the macro edits. I will work through that today and update the issue tomorrow at the latest.

**Comment by awolfe76** [comment]
Closing for now. Added a `2018` label so we can reference it later.
